selene/schema/scan_configs/mutations.py

```diff
@@ -89,33 +79,50 @@ class DeleteScanConfigsByIds(DeleteByIdsClass):
 
     Returns:
         ok (Boolean)
+    """
 
-    Example
+    class Arguments:
+        scan_config_ids = graphene.List(
+            graphene.UUID,
+            required=True,
+            name='ids',
+            description="List of UUIDs of entities to delete.",
+        )
+        ultimate = graphene.Boolean()
 
-        mutation {
-            deleteScanConfigByIds(
-                ids: ["5f8e7b31-35ea-4b43-9797-6d77f058906b"],
-                ultimate: false)
-            {
-                ok
-            }
-        }
+    ok = graphene.Boolean()
 
-        Response
-        {
-            "data": {
-                "deleteScanConfigByIds": {
-                    "ok": true
-                }
-            }
-        }
-    """
+    @staticmethod
+    @require_authentication
+    def mutate(_root, info, scan_config_ids=None, ultimate=None):
+        gmp = get_gmp(info)
 
+        filter_string = ''
+        for scan_config_id in scan_config_ids:
+            filter_string += f'uuid={str(scan_config_id)} '
+        # Get the configs via a filter. This is needed because we need to
+        # be sure that the configs we want to delete really exist. Else
+        # we might only delete some of the entities until an error
+        # interrupts the deletion process.
+        configs_xml = gmp.get_scan_configs(filter_string=filter_string)
 
-DeleteByFilterClass = create_delete_by_filter_mutation(entity_name='config')
+        configs = configs_xml.findall("config")
+        found_ids = []
+        for config in configs:
+            found_ids.append(config.get('id'))
 
+        # Configs only get deleted if all entities were found.
+        if len(scan_config_ids) != len(found_ids):
+            return DeleteScanConfigsByIds(ok=False)
```

Comment:
I really think it would be better to raise an error. I don't think we are checking the ok value of the response at the moment.

Reply:
As discussed we will leave it for now ...